### ISSUE TITLE
Should SettingOptions.txt be translated?

### ISSUE THREAD

**Issue by ingulsrud** [bobRafat/VCM-Localization#1]
The contents of SettingOptions.txt looks like a list variables, and many of the entries (but not all…) use underscores in place of space characters:

Click_Charge_Paid_By
Custom Setting

If these are to be translated, are there certain characters (such as space) that should be avoided?

**Comment by bobRafat** [comment]
Yes.the content of this file should be translated. This is the content of a db table which is in different format. Spaces should be replaced with underscore as it could not have spaces in the table strings.

**Comment by ingulsrud** [comment]
The following original English strings contain spaces, is this a problem?

Need Approval
Approval Type
Ordering View 
Custom Setting
Purchase Order (PO)

**Comment by bobRafat** [comment]
NO,They are fine, we completely sanitize strings before using them in view, so there won't be any problem.
